mkdocs.yml

```diff
@@ -254,7 +253,9 @@ nav:
       - Webhooks: "user-guide/platform-functionality/webhook.md"
   - Developer Guide:
     - "development/index.md"
-    - Jobs Developer Guide: "development/jobs.md"
+    - Migration Guides:
+      - "development/migration/index.md"
+      - Jobs Developer Guide: "development/jobs.md"
```

Comment:
Sorry I was unclear in my previous comment. We still want `Jobs Developer Guide` to be at the same level as `App Developer Guide` - my suggestion was something more like:

```
- Developer Guide:
  - "development/index.md"
  - Jobs Developer Guide:
    - "development/jobs.md"
    - Migration Guides: "development/migration/index.md"
  - App Developer Guide:
    - "development/apps/index.md"
    ...
    - Migration Guides: "development/migration/index.md"
```